integration-cli/docker_api_build_test.go

```diff
@@ -16,19 +16,9 @@ import (
 
 func (s *DockerSuite) TestBuildAPIDockerFileRemote(c *check.C) {
 	testRequires(c, NotUserNamespace)
-	var testD string
-	if testEnv.DaemonPlatform() == "windows" {
-		testD = `FROM busybox
-COPY * /tmp/
-RUN find / -name ba*
-RUN find /tmp/`
-	} else {
-		// -xdev is required because sysfs can cause EPERM
-		testD = `FROM busybox
-COPY * /tmp/
-RUN find / -xdev -name ba*
-RUN find /tmp/`
-	}
+	testD := `FROM busybox
```

Comment:
If I understand this correctly the old test expected a dockerfile to exist at `Dockerfile` and no file named `baz` to exist.

But the test fixture seems to be just a remote server with a single file at `/testD`. Where do these extra files come from?